### ISSUE TITLE
backprop_through_attack flag in mnist tutorial seems to be stoping gradients before adversarial training is defined

### ISSUE THREAD

**Issue by npapernot** [tensorflow/cleverhans#269]

**Comment by npapernot** [comment]
Closed by 1ddf7ccf952d0048807b988644068d60cf9fd55e
